### PR TITLE
feat: add log level config via env var

### LINKED ISSUE #187: Configure logging level via environment variable
**Is your feature request related to a problem? Please describe.**
Currently, log level is set upon image build. We should make this configurable via some config or env var for easier debugging

**Describe the solution you'd like**
Add some env var, LOGLEVEL=x, for configuring logging level

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by hongalex**
**What type of PR is this?**
> /kind feat

**What this PR does / Why we need it**:
Adds log level via environment variable

**Which issue(s) this PR fixes**:
Closes #187

**Special notes for your reviewer**:

### CODE REVIEW

File: internal/app/server/open_saves.go [hunk 1]
```diff
@@ -362,6 +364,7 @@ func (s *openSavesServer) insertExternalBlob(ctx context.Context, stream pb.Open
 }
 
 func (s *openSavesServer) CreateBlob(stream pb.OpenSaves_CreateBlobServer) error {
+	log.Debug("Creating blob stream\n")
```
[the remark this answers is not in the document]
Comment (hongalex) [reply internal/app/server/open_saves.go:367]: Yeah. I added another log statement below after metadata has been received.

File: cmd/server/main.go [hunk 2]
```diff
@@ -32,6 +31,7 @@ func main() {
 	defaultBucket := getEnvVarString("OPEN_SAVES_BUCKET", "")
 	defaultProject := getEnvVarString("OPEN_SAVES_PROJECT", "")
 	defaultCache := getEnvVarString("OPEN_SAVES_CACHE", "localhost:6379")
+	defaultLogLevel := getEnvVarString("LOG_LEVEL", "warn")
```
[the remark this answers is not in the document]
Comment (hongalex) [reply cmd/server/main.go]: Change to info.

File: cmd/server/main.go [hunk 3]
```diff
@@ -41,6 +41,12 @@ func main() {
 		cache   = flag.String("cache", defaultCache, "The address of the cache store instance")
 	)
 
+	ll, err := log.ParseLevel(defaultLogLevel)
+	if err != nil {
+		ll = log.DebugLevel
+	}
+	log.SetLevel(ll)
+	log.SetOutput(os.Stdout)
```
[the remark this answers is not in the document]
Comment (hongalex) [reply cmd/server/main.go]: Done. Also added flag

**Review by yuryu — APPROVED** [review]
lgtm

File: cmd/server/main.go [hunk 4]
```diff
@@ -56,6 +57,13 @@ func main() {
 		log.Fatal("missing -cache argument for cache store")
 	}
 
+	ll, err := log.ParseLevel(*logLevel)
+	if err != nil {
+		ll = log.InfoLevel
+	}
+	log.SetLevel(ll)
+	log.Infoln("Log level is: %s", ll.String())
```
Comment (yuryu) [inline cmd/server/main.go:65]: Infof?

**Review by yuryu — APPROVED** [review]

File: internal/app/server/open_saves.go [hunk 5]
```diff
@@ -378,6 +381,8 @@ func (s *openSavesServer) CreateBlob(stream pb.OpenSaves_CreateBlobServer) error
 		log.Error("CreateBlob: first message was not metadata")
 		return status.Error(codes.InvalidArgument, "The first message must be metadata.")
 	}
+	log.Debugf("Got metadata from stream: store(%s), record(%s), blob size(%s)\n",
```
Comment (yuryu) [inline internal/app/server/open_saves.go:384]: blob size(%v)

**Pull request merged**